jack-test/test/java/com/rapleaf/jack/transaction/TestTransactorMetrics.java

```diff
@@ -0,0 +1,73 @@
+package com.rapleaf.jack.transaction;
+
+import java.util.concurrent.ExecutorService;
+
+import com.google.common.base.Stopwatch;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.rapleaf.jack.IDb;
+import com.rapleaf.jack.JackTestCase;
+import com.rapleaf.jack.test_project.DatabasesImpl;
+import com.rapleaf.jack.test_project.database_1.IDatabase1;
+
+import static org.junit.Assert.assertTrue;
+
+public class TestTransactorMetrics extends JackTestCase {
+
+  private TransactorImpl.Builder<IDatabase1> transactorBuilder = new DatabasesImpl().getDatabase1Transactor();
+  private ExecutorService executorService;
+  private static final Logger LOG = LoggerFactory.getLogger(TestTransactorMetrics.class);
+  private Stopwatch stopwatch = new Stopwatch();
+
+  @Before
+  public void prepare() throws Exception {
+    transactorBuilder.get().query(IDb::deleteAll);
+    stopwatch.start();
+  }
+
+  @After
+  public void cleanup() throws Exception {
+    executorService = null;
```

Comment:
Again, executor service needs to be `shutdown`, and setting it to `null` is unnecessary.